Packs/Shodan/ReleaseNotes/1_2_3.md

```diff
@@ -3,4 +3,8 @@
 
 ##### Shodan v2
 
-- Metadata and documentation improvements.
+- Improved implementation of the **search** command
+  - Added useful details from Shodan search results to Context and War Room outputs
+    - Details about SSL/TLS certificates
+    - NTLM properties, often included in results for Remote Desktop Protocol
+  - Added new  *return_json* argument, which returns the complete Shodan search results as JSON
```

Comment:
```suggestion
  - Added useful details from Shodan search results to Context and War Room outputs.
    - Details about SSL/TLS certificates.
    - NTLM properties, often included in results for Remote Desktop Protocol.
  - Added new  *return_json* argument, which returns the complete Shodan search results as JSON.
```

Reply:
I restored the 1_2_3.md RN from master that had not been merged into my dev branch and bumped my contribution to 1.2.4.  I incorporated your suggested change in that RN.